portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx

```diff
@@ -122,7 +122,7 @@ Now that you've placed your first auction on your marketplace, someone can make
 Check out the method below, that allows a user to make a bid!
 
 ```tsx:index.ts
-market.makeAuctionListingBid({
+marketPlace.makeAuctionListingBid({
```

Comment:
```suggestion
marketplace.makeAuctionListingBid({
```

this code is not correct, you need to test the code and make sure it works.

This should be marketplace.auction.makeBid(...